packages/app/src/views/Home.vue

```diff
@@ -1,7 +1,8 @@
 <template>
-  <div id="wrapper" class="mx-auto max-w-lg h-full w-full flex flex-col">
+  <div class="app mx-auto max-w-lg h-full w-full flex flex-col">
     <header class="h-1/5 flex items-end justify-center text-4xl text-gray-800">
       <Button
+        :to="{ name: 'add-tag' }"
```

Comment:
![image](https://user-images.githubusercontent.com/40185075/104301714-40114c80-54c8-11eb-85cf-36be38703079.png)
the button is currently not working

Reply:
Good point missed that at while rebasing.

Reply:
fixed by 8ee28bb52e0a85b39cf9f7815a2edb741473e1a4